ote/src/cljs/ote/ui/table.cljs

```diff
@@ -48,11 +47,14 @@
                 (when class
                   {:class class}))
      [ui/table-header {:key (str table-id "-hdr")
-                       :style {:overflow "visible" :background-color (color :grey300)}
+                       :style {:overflow "visible"
+                               :background-color (color :grey300)
+                               :border-bottom (str "1px solid " colors/gray650 " !important")}
```

Comment:
Why only -bottom? Should it be all borders?

Reply:
Nope. Issue was only with bottom-border